docs/designers-developers/developers/backward-compatibility/deprecations.md

```diff
@@ -2,280 +2,285 @@
 
 For features included in the Gutenberg plugin, the deprecation policy is intended to support backward compatibility for two minor plugin releases, when possible. Features and code included in a stable release of WordPress are not included in this deprecation timeline, and are instead subject to the [versioning policies of the WordPress project](https://make.wordpress.org/core/handbook/about/release-cycle/version-numbering/). The current deprecations are listed below and are grouped by _the version at which they will be removed completely_. If your plugin depends on these behaviors, you must update to the recommended alternative before the noted version.
 
+## 9.7.0
```

Comment:
Pretttier triggered the update to many existing lines 😅

Reply:
I'm wondering though why the whitespace changes triggered now and not sooner...

Reply:
We never executed Prettier on `.md` files, but I wouldn't mind personally doing so.

@mkaz, how do you feel about this idea?